src/main/include/subsystems/Climber.hpp

```diff
@@ -44,7 +48,12 @@ class Climber : public SubsystemBase {
         frc3512::Constants::Climber::kTransverserPort,
         rev::CANSparkMax::MotorType::kBrushless};
 
+    rev::CANEncoder m_limiter{m_elevator,
+                              rev::CANEncoder::EncoderType::kHallSensor};
```

Comment:
Based on this,
```cpp
    /**
     * Constructs a CANEncoder.
     *
     * @param device The Spark Max to which the encoder is attached.
     * @param sensorType The encoder type for the motor: kHallEffect or kQuadrature
     * @param counts_per_rev The counts per revolution of the encoder
     */
    explicit CANEncoder(CANSparkMax& device, EncoderType sensorType = CANEncoder::EncoderType::kHallSensor, int counts_per_rev = 42);
```
you can do the following:
```suggestion
    rev::CANEncoder m_limiter{m_elevator};
```
Which do we think offers more clarity to the user? I'm on the fence about it.

Reply:
I was on the fence about it as well, with the short version being simple and the longer version providing more information to the user. I was thinking about the convenience of the longer version as the user doesn't have to take time to look in either REV Robotics' documentation or look into the CANEncoder header to find the default EncoderType, as the information is already there. However, this is my opinion. Thoughts?

Reply:
That's fair.